main.py

```
@@ -6,29 +6,22 @@
 import sentry_sdk
 import torch
 
-from fastapi import FastAPI, Request, Query, Depends, HTTPException
+from fastapi import FastAPI, Request
 from fastapi.middleware.cors import CORSMiddleware
 from fastapi.responses import JSONResponse
-from pydantic import BaseModel
 from fastapi.security import HTTPBearer
 from fastapi.openapi.utils import get_openapi
 
 from api.match import get_match_info_by_date, get_match_preview_info
```

Ruff (0.11.9)

`api.match.get_match_info_by_date` imported but unused

Remove unused import

(F401)